app/graphql/mutations/rock_and_pebbles/activate_rock_pebble_relationship.rb

```diff
@@ -0,0 +1,23 @@
+module Mutations
+  module RockAndPebbles
+    class ActivateRockPebbleRelationship < ::Mutations::BaseMutation
+      argument :id, ID, required: true
+      type Types::RockAndPebbleType
+
+      def resolve(attributes)
+        rock_and_pebble = RockAndPebble.find(attributes[:id])
+        send_pebble_confirmation_email(rock_and_pebble)
+        rock_and_pebble.update(active: true)
```

Comment:
I forget what we discussed...do we want this mutation to double for cancelling a relationship? If so then the toggle method might work well here (rock_and_pebble.toggle!(:active). We would also need an if else statement on the email message. If we had decided that the cancel relationship should be it's own mutation then disregard all of this!

Reply:
I don't remember discussing it and it's not specified on the card. Great point though! Let me ask in the channel!

Reply:
I just looked on the project board and there's another card for it! Sorry I didn't check there first. this looks great.